### DISCUSSION TITLE
Looking for new Working Group members!

### DISCUSSION THREAD

**Discussion by SteveL-MSFT** [PowerShell/PowerShell#16501] [General]
We're looking for more community members to help with our [Working Groups](https://github.com/PowerShell/PowerShell/blob/master/docs/community/working-group-definitions.md).  Requirements:

- Recognition by community as a PowerShell expert in at least one Working Group area
- Ability to engage in collaborative discussion while keeping an open mind
- Ability to follow our [Code of Conduct](https://github.com/PowerShell/PowerShell/blob/master/CODE_OF_CONDUCT.md)

Responsibilities:

- Commitment to actively participate in Working Groups on a regular basis (minimally a few hours a week)
- Triage issues and pull requests marked for their Working Group(s) and engage with other WG members to conclude on a decision (may require some investigation to understand the ask or feasibility)

If you are interested, please reply here along with the specific Working Group(s) you believe you can have an impact with.

**Comment by michaeltlombardi** [comment]
I'm interested, particularly in the DSC working group and secondarily in the Developer Experience working group.

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this and would love to have you on both the DSC and DevEx WGs

**Comment by royashbrook** [comment]
I'm interested in the Developer Experience working group.

**Reply by iSazonov** [reply]
@royashbrook Because Team will consider your candidacy please provide more information about you and your activities in PowerShell community.

**Reply by royashbrook** [reply]
I am an avid developer of small powershell modules, mostly very purpose built.. I blog about powershell sometimes and use vscode and powershell daily. Would love to help improve the powershell development experience for myself and others. My most recent community contribution was adding sql queries to the [Import-Excel](https://github.com/dfinke/ImportExcel) module.

More about me:
[github](https://github.com/royashbrook)
[blog](https://ashbrook.io)
[linkedin](https://linkedin.com/in/royashbrook)

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee appreciates your offer, but we'd like to see more activity in the repository and community before considering including you in the DevEx WG

**Comment by baderjavaid** [comment]
I'm interested, in joining DSC working group.

**Reply by iSazonov** [reply]
@baderjavaid Because Team will consider your candidacy please provide more information about you and your activities in PowerShell community.

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee appreciates your offer, but we'd like to see more activity in the repository and community before considering including you in the DSC WG

**Comment by kasini3000** [comment]
I am the founder of kasini3000,
http://gitee.com/chuanjiao10/kasini3000
This is the package of the ps-remoting to win/linux tool. It is similar to ANSIBLE, SALT. Can I set up a working group under psteam? And win some fans? Currently this project needs to be Englishized.

Based on nodelist.csv, (supports custom host list files) kasini3000 supports batches.
It should be based on kasini3000, supplemented by powershell dsc, to attract ansible and salt users to kasini3000, so that users can experience the benefits of remotely transmitting objects.

----

In addition, I suggest to open another post to collect & discuss what is the priority next year (ps7. 3)?
For example: I suggest powershell gallery to build rsync server, so that it is convenient for fans to build mirror.
https://github.com/PowerShell/PowerShellGallery/issues/164

----

By the way, I used powershell to build a better container management tool than kubernetes, but I wanted to make it into commercial software, so I had to rewrite it in GO language. At present, it is difficult for no one to invest. If it succeeds, it will be the glory of my life.When I have achieved my career, I have money. Maybe I plan to restore this software to powershell and open source.
More information & papers at:
http://gitee.com/chuanjiao10/k4t

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee did not see a formal request to join a WG, so we wish you luck in your projects

**Reply by kasini3000** [reply]
Where is the list of WGs? how to formally join?  what WG  Suggest me to join?

**Reply by iSazonov** [reply]
@kasini3000 All WG docs https://github.com/PowerShell/PowerShell/tree/master/docs/community

In PowerShell-RFC repo you can find an RFC about WG too.

**Reply by kasini3000** [reply]
i see,thank you ! @iSazonov  .

i formally request join remoting WG.Do I need to click the join button? Where is it on github? tell me click step please.
I am willing to do something for the ps community with my less ability.
I am also interested in threadjob.

**Reply by iSazonov** [reply]
@kasini3000 I see @SteveL-MSFT monitor the thread and will inform you. 
You should be recognized as active community member and expert in one or some PowerShell areas.

**Comment by gaelcolas** [comment]
I'd be interested in helping the DSC Working Group.
As I already have the DSC Community commitments, it's probably best to have someone like @michaeltlombardi though :)

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this and would love to have you in the DSC WG

**Comment by doctordns** [comment]
I would love to help with Cmdlets and Modules.

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this and would love to have you in the Cmdlets WG

**Comment by SeeminglyScience** [comment]
I can go in dev ex, engine, interactive, and/or language

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this and would love to have you in all those WGs

**Comment by jhoneill** [comment]
I'd be happy to join many of them dev-ex , engine, interactive Ux, Language, Cmdlets and modules. 
I did a lot of DSC but nothing in the last 3 years, and security and remoting aren't my strong areas.

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this, we'd like to have you start in the Cmdlets WG and can revisit adding you to others in the future

**Comment by kilasuit** [comment]
I think you need an addional few WG's that relate more around the management across the organisation (like Discussion/Issue/PR triage)

Also I think there is definitely a need for more work in the Interactive UX so that we don't just get bogged down in developer experience & as such I'd like to put myself forward for that WG & as there is some overlap with the Cmdlets and Modules WG I would also like to be considered for that one too

**Reply by vexx32** [reply]
I think the WG doc mentions specifically that maintainers are deliberately not a WG, but I'd be inclined to agree that it would be a good idea.

**Reply by kilasuit** [reply]
There's a difference between maintainers and general helpers, which is why I think an additional WG just for helpers isn't a bad idea.

**Reply by iSazonov** [reply]
WG is a group of experts. We need _active_ WGs to have good progress. If the "helper" is an expert in an area the "helper" will be a member of one or some appropriate WGs.

**Reply by kilasuit** [reply]
@iSazonov - I;ve been less active in the community in recent years for a number of reasons, including volunteering vaccinating last & this year and living somewhere I hated. I intend to be spending much more time in 2022 being active in the community again, hence my point for consideration

@vexx32  my point was more theres a need on cross org helpers not just those working on a single repo in the org

**Reply by iSazonov** [reply]
@kilasuit We need more code reviewers to speed up PRs and we need more experts to triage issues by WGs - welcome!

**Reply by kilasuit** [reply]
I can't do the first right now but can definitely more of the second to hopefully help free up time for those who can do the first better than I can

**Reply by SteveL-MSFT** [reply]
@PowerShell/powershell-committee reviewed this and would love to have you in the Interactive UX WG to start with

**Comment by iSazonov** [comment]
Are there heroes for WG-Engine-Providers and WG-Engine-Format? Huge work is waiting you :-)

**Comment by SteveL-MSFT** [comment]
For those who recently accepted, I'll send out invites to the Discord server used by the WGs tomorrow or by Monday.  They will be sent as DMs to your Twitter accounts.

**Comment by IISResetMe** [comment]
I'm interested in helping the Engine and Language Working Groups :)

**Reply by kilasuit** [reply]
I'd be more than happy to support this

**Reply by SteveL-MSFT** [reply]
Forgot to inform that @IISResetMe has been added

**Comment by Tr4yw4y** [comment]
Good job

**Comment by powercode** [comment]
I'm interested in helping the Developer Experience, Engine, and/or Language Working Groups

**Reply by kilasuit** [reply]
@SteveL-MSFT if @powercode is still interested then I think they'd be a good addition to any of the working groups they've expressed interest in.

**Reply by SteveL-MSFT** [reply]
@powercode sorry for the late reply, but we would love to have you in the DevEx and Engine WGs to start with.  Please ack if you are still interested.

**Reply by powercode** [reply]
@SteveL-MSFT  Ack

**Reply by powercode** [reply]
@SteveL-MSFT What's the next step?

**Reply by dkattan** [reply]
@SteveL-MSFT Also wondering the same

**Reply by SeeminglyScience** [reply]
Hey @powercode so sorry I kept trying to find this thread again and getting distracted. Is there somewhere I can DM you? The Engine WG uses discord for meetings, if you already have discord shoot me a line (`seeminglyscience` there as well) I'll get you the invite

**Comment by FriedrichWeinmann** [comment]
Might not be the most active member possible, but would be willing to try and pitch in with the "Interactive UX" WG

**Reply by kilasuit** [reply]
@theJasonHelmick I think it would be good to add @FriedrichWeinmann to the Interactive-UX

**Reply by theJasonHelmick** [reply]
Thank you @kilasuit and @FriedrichWeinmann -- I will send out a new meeting invitation beginning in January -- looking forward to seeing you Fred when you can join!!!

**Comment by dkattan** [comment]
I would like to join the Developer Experience, Engine, and/or Language Working Groups. I write an application called ImmyBot used by over 500 Microsoft partners that integrates very deeply with PowerShell and PowerShellEditorServices for a web based debugger. ImmyBot makes heavy use of Constrained Language most to provide a trusted execution context that persists reboots while sending scriptblocks to remote machines similar to PowerShell Workflows. I have found many edge cases related to Constrained Language mode and lack of FileSystem provider and I think I could be a great addition to these teams.

**Reply by dkattan** [reply]
So I guess that’s a no?

**Reply by dkattan** [reply]
@SteveL-MSFT Bump

**Comment by theJasonHelmick** [comment]
Hi @dkattan and others in this thread.  We are working on a new template to help us keep track of requests and allow the Working Groups to review and contact folks.  I expect we will announce the process and template at the next community call.  I look forward to helping ya'll get those requests answered :)
